src/main.jsx

```diff
@@ -2,29 +2,34 @@ import React from 'react'
 import ReactDOM from 'react-dom/client'
 import { CommentSection } from './CommentSection'
 
-// Create a global function to initialize the comments
+// Create a function to initialize the comments
 console.log('Initializing Bluesky comments');
-window.initBlueskyComments = (elementId, uri) => {
+const initBlueskyComments = async (elementId, author) => {
   const element = document.getElementById(elementId)
   if (element) {
-    // Ensure URI is properly formatted
-    let formattedUri = uri;
-    if (!uri.startsWith('at://')) {
-      // If it's a bsky.app URL, convert it
-      if (uri.includes('bsky.app/profile/')) {
-        const match = uri.match(/profile\/([\w.]+)\/post\/([\w]+)/);
-        if (match) {
-          const [, did, postId] = match;
-          formattedUri = `at://${did}/app.bsky.feed.post/${postId}`;
-        }
+    const currentUrl = window.location.href;
+    const apiUrl = `https://public.api.bsky.app/xrpc/app.bsky.feed.searchPosts?q=*&url=${encodeURIComponent(currentUrl)}&author=${author}`;
```

Comment:
nit: I think you can add `?limit=1` to only return a single result, which would be slightly more efficient.

Reply:
Oh nvm just re-read hte PR description and saw you're gonna update this